README.md

```diff
@@ -87,6 +99,33 @@ configuration Sample_xDnsServerZoneTransfer_TransferToAnyServer
 Sample_xDnsServerZoneTransfer_TransferToAnyServer -DnsZoneName 'demo.contoso.com' -TransferType 'Any'
 ```
 
+### Configuring a Primary Standalone DNS Zone
+
+```powershell
+configuration Sample_xDnsServerPrimaryZone
+{
+    param
+    (
+        [Parameter(Mandatory)]
+        [String]$ZoneName,
+        [Parameter()] [ValidateNotNullOrEmpty()]
+        [String]$ZoneFile = "$ZoneName.dns",
```

Comment:
I think we should either open an issue or fix it now.  changing a parameter from required to optional is not breaking so it doesn't have to be done in this PR.